test/data/minimal_fast/execution-environment.yml

```diff
@@ -0,0 +1,18 @@
+version: 3
+
+additional_build_steps:
+  prepend_base:
+  # a quick value we can check for cached output
+  - RUN echo "$(echo hi) $(echo mom)"
+
+images:
+  base_image:
+    name: quay.io/centos/centos:stream9
```

Comment:
FWIW, I'd really like to move away from using this image due to stability issues. Doesn't have to happen now, but just FYI.